config/application.rb

```diff
@@ -330,13 +330,9 @@ def dynflow
     def init_dynflow
       dynflow.eager_load_actions!
 
-      unless dynflow.config.lazy_initialization
-        if defined?(PhusionPassenger)
-          PhusionPassenger.on_event(:starting_worker_process) do |forked|
-            dynflow.initialize! if forked
-          end
-        else
-          dynflow.initialize!
+      if !dynflow.config.lazy_initialization && defined?(PhusionPassenger)
```

Comment:
Nit: I think this is easier to read:

```suggestion
      if defined?(PhusionPassenger) && !dynflow.config.lazy_initialization
```

Reply:
Indeed. Thanks